spec/rails_helper.rb

```diff
@@ -18,7 +18,7 @@
   Capybara.javascript_driver = :selenium_chrome_headless
 end
 
-Capybara.default_max_wait_time = 5
+Capybara.default_max_wait_time = 3
```

Comment:
Initially I assumed that this wait time would help with our failures because there were a lot of timing issues with us comparing a piece of text in the wrong body. I increased this time to 10 seconds but it made no impact on improving the failures. It seems like the issue was that we were waiting for the text to appear on the wrong page. By relying on sleeps between pages we can ensure that we don't poll too early. It made me think that this default_max_wait_time was actually probably adding a lot of unnecessary time to our tests and didn't need to be so high.